### PR TITLE
Adding contributor list to README.md

### PR DESCRIPTION
**Pull request by EhrenfeldN**
Issue #28 :)

### CODE REVIEW

**Comment by pyymenta** [comment]
thank you about your contribuition! I have some points:

File: README.md [hunk 1]
```diff
@@ -43,3 +43,34 @@ yarn build
...
+</a>
+&nbsp; 
+<a href="https://github.com/IJustDev">
+  <img src="https://avatars0.githubusercontent.com/u/43793996?s=64&v=44">
+</a>
+&nbsp; 
+<a href="https://github.com/marcotgs">
+  <img src="https://avatars1.githubusercontent.com/u/20916344?s=64&v=4">
+</a>
+&nbsp;
```
Comment (pyymenta) [inline README.md]: There is a way to generate it automatically?

File: README.md [hunk 2]
```diff
@@ -43,3 +43,34 @@ yarn build
 ```sh
 yarn test
 ```
+
+## Contributor
```
Comment (pyymenta) [inline README.md]: Typo here, it should be in plural form

**Comment by ianmuchina** [comment]
You could try this 
https://contributors-img.web.app/preview?repo=pyymenta%2Fspacecraft-cli

**Comment by pyymenta** [comment]
> You could try this
> https://contributors-img.web.app/preview?repo=pyymenta%2Fspacecraft-cli

Ohh, love it! @EhrenfeldN what do you think? 

Thank you @IanMuchina !

**Comment by EhrenfeldN** [comment]
That's  look great @pyymenta ! I add the modification and the plural form for "Contributor" :)

Thx @IanMuchina !

**Comment by pyymenta** [comment]
> That's look great @pyymenta ! I add the modification and the plural form for "Contributor" :)
> 
> Thx @IanMuchina !

Very good! Looks good to me. But we have some conflicts with master/README.md

Could you fix it?

**Comment by EhrenfeldN** [comment]
Fixed @pyymenta ! ;)

**Review by pyymenta — APPROVED** [review]
Awesome!! :rocket: :rocket: :rocket:

**Comment by pyymenta** [comment]
Great job! I'll merge in! Thanks :heart:

**Pull request merged**